### PR TITLE
update checksum of the source tarball after including updated release…

### PR DESCRIPTION
**Pull request by nandedamana**
… info in the appstream metainfo

### CODE REVIEW

**Comment by nandedamana** [comment]
bot, build

File: in.co.nandakumar.vara.yml [hunk 1]
```diff
@@ -21,4 +21,4 @@ modules:
     sources:
       - type: archive
         url: https://nandakumar.co.in/software/vara/downloads/0.1/vara-0.1.240602.tar.gz
-        sha256: e40909b7d62de08ec6b891ad9b9bde515caf3a9345ae081ecc6362da97f64b0f
+        sha256: fac61f516ab6cda010cff8393d276ce5307c3378772ce7ab10cb676d77b57573
```
Comment (hfiguiere) [inline in.co.nandakumar.vara.yml:24]: generating a new tarball without changing the name is a like retagging: a BAD practice.

**Comment by nandedamana** [comment]
On 02/06/24 18:01, Hubert Figuière wrote:
> generating a new tarball without changing the name is a like 
> retagging: a BAD practice.

I know, but that'd break other things, and I do not want to spend this 
much time and energy for something in beta stage. BTW, I think I did it 
twice today, once due to a mistake that I made, and once due to the 
appstream metainfo getting validation error during remote build which 
was validating fine locally.

I've been following certain bad practices here due to priorities. I hope 
I can do better once I automate more things.

**Comment by nandedamana** [comment]
@hfiguiere Could you please help me understand why the new version is not showing up on Flathub even after three hours? Is it because of the above-mentioned bad practice (I hope not because the changes are already merged to the master)? Sorry for disturbing, but this is the first time I'm releasing a Flathub update and I've gone through [the doc](https://docs.flathub.org/docs/for-app-authors/maintenance/)

**Comment by nandedamana** [comment]
The new version is online now. This is the part from the doc that confused me: "The build is published (i.e. signed and imported) into the Flathub Flatpak repo manually (via the web ui) or **automatically after three hours**." In my case, the new version wasn't available when I checked after seven hours (yesterday). It was available today morning. If this is a usual delay, maybe something like "(it may take more than three hours)" should be added to the doc to avoid maintainers worrying there's something wrong with their build?

**Pull request merged**